src/main/webapp/app/shared/firebase/input/RealtimeBasicInput.tsx

```diff
@@ -139,6 +139,12 @@ const RealtimeBasicInput: React.FunctionComponent<IRealtimeBasicInput> = (props:
   const inputComponent = (
     <>
       <Input
+        innerRef={inputRef => {
+          if (inputRef && type === RealtimeInputType.TEXTAREA) {
+            inputRef.style.height = 'auto';
+            inputRef.style.height = `${inputRef.scrollHeight}px`;
```

Comment:
We overwrite the height twice?

Reply:
The first overwrite is for the case when the text box needs to shrink. When a row is deleted, the scroll height will not automatically change. That set to `auto` causes it to shrink to fit it's content.

There second one is for when the box grows. In that case the scroll height will automatically increase, and so I just need to set the height to the scroll height.

Reply:
Interesting, so basically after assigning auto, the inputref height is updated right away so you can get the height in the second assignment?